.github/workflows/tests-docs.yaml

```diff
@@ -34,4 +34,6 @@ jobs:
           uv sync --group tests --extra all --extra a2a
 
       - name: Run Documentation tests
-        run: pytest -v tests/docs
+        # The env key is necessary for docs involving OTel
+        # see https://protobuf.dev/news/2022-05-06/#python-updates
+        run: PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=python pytest -v tests/docs
```

Comment:
Can this be set in the docs test py file instead of in the github action?

Reply:
I'll check. I tried setting it in the notebook itself, but it was too late in the process :-/

Reply:
https://github.com/mozilla-ai/any-agent/pull/513

Reply:
@javiermtorres I don't understand the problem and/or the link added as reference (I didn't find much valuable information there).

Could it be just a version mismatch of the protobuf dependency in a2a? I see they have just released https://github.com/a2aproject/a2a-python/releases/tag/v0.2.9